### PR TITLE
Vale add

### PR DESCRIPTION
**Pull request by CoopGorm**

### CODE REVIEW

**Review by CRCOOP — DISMISSED** [review]

File: Coop/styles/Vocab/CoopAbbreviations/accept.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 AAD
-Ajax
+[aA]jax
```
Comment (nhhagen) [inline Coop/styles/Vocab/CoopAbbreviations/accept.txt:2]: Why do we need the alternate spelling?

File: Coop/styles/Vocab/CoopAbbreviations/accept.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
 ABAP
+[aA]bap
```
Comment (nhhagen) [inline Coop/styles/Vocab/CoopAbbreviations/accept.txt:4]: Why do we need the alternate spelling?

File: Coop/styles/Vocab/CoopAbbreviations/accept.txt (hunk 1 above)
Comment (CoopGorm) [reply Coop/styles/Vocab/CoopAbbreviations/accept.txt:2]: I got an error with ajax in the SAP neptune guidelines.

Comment (nhhagen) [reply Coop/styles/Vocab/CoopAbbreviations/accept.txt:2]: I took a look at the error in the build, in both of the Ajax cases I would fix the error by replacing ajax with Ajax, if you are referencing a specific directory, file or command I recommend using monospace.

You get monospace like this:

```markdown
`ajax`
```

File: Coop/styles/Vocab/CoopAbbreviations/accept.txt (hunk 2 above)
Comment (nhhagen) [reply Coop/styles/Vocab/CoopAbbreviations/accept.txt:4]: As far as I have been able to figure out APAB is written with capital letters.

Comment (CoopGorm) [reply Coop/styles/Vocab/CoopAbbreviations/accept.txt:4]: We do both.

Comment (nhhagen) [reply Coop/styles/Vocab/CoopAbbreviations/accept.txt:4]: Ok

**Review by nhhagen — CHANGES_REQUESTED** [review]
Let's do the ABAP change and revert the Ajax change. We have spent a significant amount of time getting these lists objectively correct to increase the quality of our documentation.

File: Coop/styles/Vocab/CoopAbbreviations/accept.txt (hunk 1 above)
Comment (nhhagen) [reply Coop/styles/Vocab/CoopAbbreviations/accept.txt:2]: https://developer.mozilla.org/en-US/docs/Web/Guide/AJAX

Comment (CoopGorm) [reply Coop/styles/Vocab/CoopAbbreviations/accept.txt]: Ok :-)

**Review by nhhagen — DISMISSED** [review]

**Review by CRCOOP — APPROVED** [review]

**Comment by nhhagen** [comment]
Let's close the PR then?

**Comment by ianrobrien** [comment]
Closing as there is no diff.

**Pull request closed without merging**